README.md

```diff
@@ -101,7 +101,7 @@ Additionally, we would like to thank the [Amethyst](https://github.com/amethyst/
 Bevy is free and open source! All code in this repository is dual-licensed under either:
 
 * MIT License ([LICENSE-MIT](LICENSE-MIT) or [http://opensource.org/licenses/MIT](http://opensource.org/licenses/MIT))
-* Apache License, Version 2.0 ([LICENSE-APACHE](LICENSE-APACHE) or [http://www.apache.org/licenses/LICENSE-2.0](http://www.apache.org/licenses/LICENSE-2.0))
+* Apache License Version 2.0 ([LICENSE-APACHE](LICENSE-APACHE) or [http://www.apache.org/licenses/LICENSE-2.0](http://www.apache.org/licenses/LICENSE-2.0))
```

Comment:
The official name is "Apache License, Version 2.0", not "Apache License Version 2.0"

Reply:
> The official name is "Apache License, Version 2.0", not "Apache License Version 2.0"

oh, my bad